# (PART\*) Student Activity {-}

# Explore the Epigenome

## Launch the Genome Browser Gateway

Go to [`https://genome.ucsc.edu/cgi-bin/hgGateway`](https://genome.ucsc.edu/cgi-bin/hgGateway). Ensure "Human" is selected under "Browse/Select Species".  

```{r, fig.align='center', echo = FALSE, fig.alt= "Landing page for the UCSC Genome browser Gateway.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_2")
```

You will be working from the Human Assembly **March 2006 (NCBI/hg18)**. Make sure this is selected under the "Find Position" dropdown menu. Enter **HOXA7** in Position/Search Term and select GO. 

```{r, fig.align='center', echo = FALSE, fig.alt= "March 2006 assembly is highlighted on the Find Position dropdown menu.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_7")
```

Your Genome Browser should now look like this:

```{r, fig.align='center', echo = FALSE, fig.alt= "March 2006 assembly results for the HOXA7 locus.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_22")
```

## Clean up Visual Settings

You’ll now use the following directions to remove the default visual settings.

Scroll below to different menu options shown. 

* Under the "Mapping and Sequence" tab, Base Position should be set to **dense**. 
* Under the "Genes and Gene Predictions" tab,  RefSeq Genes should be set to **pack**.  
* All other features should be marked as **hide**. You will need to go through each of the menu options to make sure features are marked as “hide”. Please note you will need to open each menu option by select the  “+” to ensure all tracks are turned off (marked as hide). 

```{r, fig.align='center', echo = FALSE, fig.alt= "The Base Position menu has been set to 'dense'.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_29")
```
```{r, fig.align='center', echo = FALSE, fig.alt= "The RefSeq Genes menu has been set to 'pack'.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_33")
```

At the end of the browser page, select "refresh". 

```{r, fig.align='center', echo = FALSE, fig.alt= "The refresh button at the bottom of the page is highlighted.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_41")
```

Your browser should look like the image below, where only HOXA7 is shown. 

```{r, fig.align='center', echo = FALSE, fig.alt= "The genome is zoomed in to the HOXA7 gene.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_48")
```

## Zooming Out

Zoom out your display by 10x, 3x or 1.5x to see HOXA1-HOXA13 genes on your browser. The zoom out value you’ll select will be dependent on your computer display screen. You may need to use the zoom in option if you’ve zoomed out the display beyond the HOXA1-HOXA13 genes. Zoom buttons can be clicked more than once.

```{r, fig.align='center', echo = FALSE, fig.alt= "The zoom in and out buttons are highlighted", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_54")
```

Ideally your browser should like the image below. 

```{r, fig.align='center', echo = FALSE, fig.alt= "HOXA1 through HOXA3 loci are now visible on the genome.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_59")
```

Instead of zooming in and out, you can always manually enter your chromosome region of choice. For this exercise, try entering the following region: **chr7:27,087,367-27,235,317**. Click "go". 

```{r, fig.align='center', echo = FALSE, fig.alt= "The specific chromosome locus has been typed into the search bar.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g15625333d53_0_65")
```

<div class = "notice">
You could also enter **chr7:27,087,367-27,235,317** in the search bar at the start of this exercise instead of HOXA7. 
</div>

## Visualizing Histone Marks

Now we’ll visualize histone marks to show that the same chromosome locus can have drastically different histone patterns.

Scroll down to the "Regulation" menu and click the **Broad Histone** track hyperlink.

```{r, fig.align='center', echo = FALSE, fig.alt= "The Broad Histone track hyperlink is highlighted.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g152864d54a7_0_0")
```

Ensure you have selected **full** under the "Peaks" menu.

```{r, fig.align='center', echo = FALSE, fig.alt= "Full is selected under the Peaks dropdown menu.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g152864d54a7_1_3")
```

Scroll down and deselect all boxes.

```{r, fig.align='center', echo = FALSE, fig.alt= "Cell line and antibody combinations have been deselected.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g152864d54a7_1_8")
```

Select **H3K4me3** and **H3K27me3** marks for **H1-hESC** and **NHLF**. To learn more about these options you can click on their blue hyperlinks for more information. 

```{r, fig.align='center', echo = FALSE, fig.alt= "Selected cell line and antibody combinations have been highlighted.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g152864d54a7_0_9")
```

<div class = "notice">
H3K4me3 is a known histone mark on Histone 3 (H3). It occurs on lysine 4 (K4) and has three methyl groups (me3) that indicate active gene expression. In other words, the DNA is open and ready to be accessed.

```{r, fig.align='center', echo = FALSE, fig.alt= "Histone H3 (tri methyl K4). Marks promoters that are active or poised to be activated.", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g152864d54a7_1_18")
```

H3K27me3 is a known histone mark on on Histone 3 (H3). It occurs on lysine 27 (K27) and has three methyl groups (me3) that indicate inactive gene expression. In other words, the DNA is closed off and cannot be accessed.

```{r, fig.align='center', echo = FALSE, fig.alt= "Histone H3 (tri-methyl K27). Marks promoters that are silenced by Polycomb proteins in a given lineage; large domains are found at inactive developmental loci", out.width = '100%'}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B7y_jm8GuHZMUvTU1TFdW4nqQQV6nOT9XuFwgSflXz4/edit#slide=id.g152864d54a7_1_22")
```
</div>